sefaz/accesskey/validation_test.go

```diff
@@ -13,9 +13,15 @@ func TestCheck_ShouldReturnNilForValidAccessKeys(t *testing.T) {
 	err := Check(key)
 
 	assert.Nil(t, err)
+
+	key = AccessKey("35210501180169000133551210501100013022024517")
```

Comment:
Nit: I think it would be better to have more than one success case in the unit test